src/functions/template-tags/multiday.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Calendar Multiday Functions
+ *
+ * Display functions (template-tags) for use in WordPress templates.
+ */
+
+// Don't load directly!
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+	return;
+}
+
+
+/**
+ * Used in the month loop.
+ * Outputs classes for the multiday event (article).
+ *
+ * @since TBD
+ *
+ * @param WP_Post $event            An event post object with event-specific properties added from the the `tribe_get_event`
+ *                                  function.
+ * @param string  $day_date         The `Y-m-d` date of the day currently being displayed.
+ * @param bool    $is_start_of_week Whether the current grid day being rendered is the first day of the week or not.
+ * @param string  $today_date       Today's date in the `Y-m-d` format.
+ *
+ * @return array $classes The classes to add to the multiday event.
+ */
+function tribe_events_multiday_classes( $event, $day_date, $is_start_of_week, $today_date ) {
```

Comment:
Move this to `src/Tribe/Views/V2/functions/classes.php`.
Create the file above and namespace in `Tribe\Events\Views\V2`.
Add some tests to cover the output in `views_wpunit`.
Load the new file [here](https://github.com/moderntribe/the-events-calendar/blob/0f273ff7ec093b9cf1bf2d7a3b043be5d9f1b6b5/src/Tribe/Views/V2/Service_Provider.php#L30)